api/core/config.py

```diff
@@ -29,10 +35,9 @@ class Settings:
         f"{POSTGRES_SERVER}:{POSTGRES_PORT}/{POSTGRES_DB}?sslmode={SSL_MODE}"
     )
 
-    SECRET_KEY = os.getenv("SECRET_KEY")
```

Comment:
What was the problem with getenv (just curious)?

Reply:
I'm just using `os.environ` everywhere and getting that data from `.env` file